docker/jenkins-scripts/smoke-test.sh

```diff
@@ -78,64 +89,77 @@ try:
+            time.sleep(10)
 
             # Launch the candlepin container:
             output = run_command("docker run -P -d -e \"YUM_REPO=%s\" --link %s:db %s"
                 % (cp_repo_url, db_container_name, image_name))
             server_container_id = output[-1]
+            print "Candlepin container: %s" % db_container_id
             time.sleep(3)
 
-            print "Server container: %s" % server_container_id
-            print "Docker ps: %s" % run_command("docker ps")
-
             # Determine the port used by the CP server...
-            output = run_command("docker port %s 8443" % server_container_id)
-            port = output[0].split(':')[1]
-            print "Candlepin container port: %s" % port
-
-            # Wait for it to start...
-            print "Containers started successfully. Waiting for Candlepin to start..."
-
-            status_url = "https://localhost:%s/candlepin/status" % port
-            start_time = time.time()
-            remaining = max_wait_time
-            response_received = False
-
-            while (remaining > 0):
-                try:
-                    response = urllib2.urlopen(status_url, None, remaining)
-
-                    response_received = True
-                    code = response.getcode()
-                    message = response.read()
-                    response.close()
-
-                    print "Response received (code: %s)\n%s\n" % (code, message)
-                    if response.getcode() == 200:
-                        status = json.loads(message)
-
-                        # TODO: Maybe check some of the fields on status to verify it?
-                        success = True
-                        print "Received a valid response from the Candlepin server in %.1fs" % \
-                            (time.time() - start_time)
-                    else:
-                        print "Invalid response received"
-
-                    break
-
-                except URLError as e:
-                    # We'll be expecting tons of errors while we wait for CP to start. So, we'll
-                    # just silently ignore them and hope for the best.
-                    pass
-
-                time.sleep(1)
-                remaining = max_wait_time - (time.time() - start_time)
-
-            if not response_received:
-                print "Failed to receive a response in %.1fs" % (time.time() - start_time)
-                print "Candlepin container log:"
-                run_command("docker logs --tail=%s %s" % (max_log_lines, server_container_id), False, True)
+            regex = re.compile(".*:(\\d+)\\s*\\Z")
+            output = run_command("docker port %s 8443/tcp" % server_container_id)
+            match = regex.match(output[0])
+
+            if match:
+                port = match.group(1)
+
+                # Wait for it to start...
+                print "Containers started successfully using port %s. Waiting for Candlepin to start..." % port
```

Comment:
Looks like your verbage change got clobbered here. If you care enough to change it back, should also correct the port message to be less awkward.